tests/model_registry/conftest.py

```diff
@@ -378,3 +402,13 @@ def fail_if_missing_authorino_operator(admin_client: DynamicClient) -> None:
 
     if not subscription_found:
         pytest.fail(f"Could not find Subscription for {operator_name} in openshift-operators namespace")
+
+
+@pytest.fixture(scope="session", autouse=True)
+def init_model_registry_namespace(admin_client: DynamicClient) -> None:
+    """Initialize MR_NAMESPACE with the admin client.
+
+    This fixture runs automatically at session start to ensure MR_NAMESPACE
+    is properly initialized before any tests run.
+    """
+    init_mr_namespace(admin_client=admin_client)
```

Comment:
We should only do this if MR is enabled by default.

Reply:
registriesNamespace is defined either way